### PR TITLE
feat(1820): Group events together

### PR DESCRIPTION
**Pull request by OronNadiv**
## Context

We want to be able to group jobs with the same commit id in order to reduce the clutter.

## Objective

Group all jobs with the same commit id.  Allow this group to expand and collapse.

image

## References

https://github.com/screwdriver-cd/screwdriver/issues/1820

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by vnugopal** [comment]
@OronNadiv  should that be "Expand". I see "Expend" in there

**Comment by OronNadiv** [comment]
@vengopal good catch.  I fixed it.

**Review by jithine — DISMISSED** [review]

**Review by jithine — DISMISSED** [review]

File: app/components/pipeline-events-row-group/component.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  }),
+  isExpanded: computed('expandedEventsGroup', 'events.[]', {
+    get() {
+      const expandedGroups = get(this, 'expandedEventsGroup');
+      const { groupEventId } = this.events[0];
+
+      return !!expandedGroups[groupEventId];
+    }
+  }),
+  init() {
```
Comment (adong) [inline app/components/pipeline-events-row-group/component.js:18]: `init()` is not needed if no actual overloading codes.

File: app/components/pipeline-events-row-group/component.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { computed, get, set } from '@ember/object';
+import Component from '@ember/component';
+
+export default Component.extend({
+  events: computed('events.[]', {
```
Comment (adong) [inline app/components/pipeline-events-row-group/component.js:5]: I think this events is not needed, just `events: []` ought to be okay.

**Review by adong — APPROVED** [review]
I checked out this branch and tested locally, looks fine to me. 
Collapsed: 
image

Expanded:
image

With minor code comment.

**Review by adong — APPROVED** [review]

**Pull request merged**